packages/shared_preferences/shared_preferences_foundation/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -197,46 +209,27 @@
 			productReference = 97C146EE1CF9000F007C117D /* Runner.app */;
 			productType = "com.apple.product-type.application";
 		};
-		F76AC2052669B6AE0040C8BC /* RunnerTests */ = {
-			isa = PBXNativeTarget;
-			buildConfigurationList = F76AC20F2669B6AE0040C8BC /* Build configuration list for PBXNativeTarget "RunnerTests" */;
-			buildPhases = (
-				DB9B98025BDEFED85B1B62A7 /* [CP] Check Pods Manifest.lock */,
-				F76AC2022669B6AE0040C8BC /* Sources */,
-				F76AC2032669B6AE0040C8BC /* Frameworks */,
-				F76AC2042669B6AE0040C8BC /* Resources */,
-			);
-			buildRules = (
-			);
-			dependencies = (
-				F76AC20C2669B6AE0040C8BC /* PBXTargetDependency */,
-			);
-			name = RunnerTests;
-			productName = RunnerTests;
-			productReference = F76AC2062669B6AE0040C8BC /* RunnerTests.xctest */;
-			productType = "com.apple.product-type.bundle.unit-test";
-		};
 /* End PBXNativeTarget section */
 
 /* Begin PBXProject section */
 		97C146E61CF9000F007C117D /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 1100;
-				ORGANIZATIONNAME = "The Flutter Authors";
+				LastUpgradeCheck = 1300;
+				ORGANIZATIONNAME = "";
```

Comment:
I'm not sure if it's consistent but `The Flutter Authors` makes sense for example apps. Not important though.